.github/workflows/publish-test-result.yml

```diff
@@ -0,0 +1,59 @@
+name: Publish Tests Result
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+
+jobs:
+  unit-tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Run unit tests and capture results
+        id: unit-test-results
+        run: |
+          yarn test --json --outputFile=test-results.json
+          cat test-results.json
```

Comment:
Do we need this line?

Reply:
Oh yeah, we don't need it. I forgot to remove it when I was using it for testing. Thanks for reminding me